nixos/modules/services/games/asf.nix

```diff
@@ -174,13 +173,17 @@ in
         wantedBy = [ "multi-user.target" ];
 
         serviceConfig = mkMerge [
-          (mkIf (cfg.dataDir == "/var/lib/asf") { StateDirectory = "asf"; })
+          (mkIf (cfg.dataDir == "/var/lib/asf") {
+            StateDirectory = "asf";
+            StateDirectoryMode = "700";
```

Comment:
I am not sure if this should be 700 or 750 but I am really sure that the last bit must be a 0.

Reply:
the last octet ;)